aten/src/ATen/Parallel.h

```diff
@@ -29,7 +29,7 @@ TORCH_API bool in_parallel_region();
 namespace internal {
 
 // Initialise num_threads lazily at first parallel call
-inline TORCH_API void lazy_init_num_threads() {
+inline void lazy_init_num_threads() {
```

Comment:
The change seems OK in a sense that it makes the code compile. Albeit I am not so sure if it won't break the DLL interface when the **`__declspec(dllexport)`** would be missing. According to my search through the codebase, it does not seem that this function is actually used accross the DLL interface but my knowledge here is limited.

cc @kit1980